specs/electra/beacon-chain.md

```diff
@@ -1090,6 +1101,20 @@ def process_withdrawals(state: BeaconState, payload: ExecutionPayload) -> None:
 
 #### Execution payload
 
+##### New `compute_execution_requests_hash`
+
+*Note*: Computes commitment to the execution layer triggered requests data.
+The computation algorithm is defined by [EIP-7685](https://eips.ethereum.org/EIPS/eip-7685).
+
+```python
+def compute_execution_requests_hash(execution_requests: ExecutionRequests) -> Hash32:
+    deposit_bytes = DEPOSIT_REQUEST_TYPE + serialize(execution_requests.deposits)
+    withdrawal_bytes = WITHDRAWAL_REQUEST_TYPE + serialize(execution_requests.withdrawals)
+    consolidation_bytes = CONSOLIDATION_REQUEST_TYPE + serialize(execution_requests.consolidations)
+
+    return hash(deposit_bytes + withdrawal_bytes + consolidation_bytes)
```

Comment:
it looks like we want to hash each input to the return value here, so its `hash(hash(..)+hash(..)..)`

Reply:
Yes! That has been updated on the EIP.